libs/libpahomqtt/Makefile

```diff
@@ -0,0 +1,40 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libpahomqtt
+PKG_VERSION:=1.3.9
+PKG_RELEASE:=1
+PKG_SOURCE_PROTO:=git
+
+PKG_SOURCE_URL:=git://github.com/eclipse/paho.mqtt.c.git
+PKG_SOURCE_VERSION:=3b7ae6348bc917d42c04efa962e4868c09bbde9f
+PKG_MIRROR_HASH:=
```

Comment:
```suggestion
PKG_NAME:=libpaho-mqtt-c
PKG_VERSION:=1.3.9
PKG_RELEASE=1

PKG_SOURCE:=v$(PKG_VERSION).tar.gz
PKG_SOURCE_URL:=https://codeload.github.com/eclipse/paho.mqtt.c/tar.gz/v$(PKG_VERSION)?
PKG_HASH:=386c9b5fa1cf6d0d516db12d57fd8f6a410dd0fdc5e9a2da870aae437a2535ed
PKG_BUILD_DIR:=$(BUILD_DIR)/paho.mqtt.c-$(PKG_VERSION)

```

Reply:
I hope to get the source code over git protocol.

Reply:
What's the reason you want to use source code instead of tagged release?

Reply:
You are right. I change the source of the source code from git to tagged release.
Thank you.